### PR TITLE
git round trip working... sorta

### PR DESCRIPTION
**Pull request by kzantow**
# Description

See [JENKINS-XXXXX](https://issues.jenkins-ci.org/browse/JENKINS-XXXXX).

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

**Comment by i386** [comment]
So good Keith 😍 I'm chipping away at an update to that wiki page you created

**Comment by michaelneale** [comment]
Seems to work well - editor PR has my comment. ATH happy with this? 

Would be nice to have ATH cover creation and editor to increase confidence of changes like this.

**Comment by kzantow** [comment]
Well, this needs 2 designs before it is ready, I think:
1) a design for 'upgrading credentials'
2) using the git flow actually prompts somewhere to go to the editor

**Comment by michaelneale** [comment]
on point #2 so you mean the creation flow needs to dump you in the editor?

**Comment by kzantow** [comment]
@michaelneale either dump you in the editor, or maybe better, the empty state needs to guide you to it with a button or something

File: blueocean-git-pipeline/src/main/java/io/jenkins/blueocean/blueocean_git_pipeline/GitReadSaveService.java [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+            content.getString("message"),
+            content.getString("sourceBranch"),
+            content.getString("path"),
+            Base64.decode(content.getString("base64Data"))
+        );
+    }
+
+    @Override
+    public Object getContent(StaplerRequest req, Item item) {
+        GitReadSaveRequest r = null;
```
Comment (i386) [inline blueocean-git-pipeline/src/main/java/io/jenkins/blueocean/blueocean_git_pipeline/GitReadSaveService.java]: make this a `Closable`

so `using(GitReadSaveRequest req = makeSaveRequest(item, req)`

**Comment by kzantow** [comment]
Superseded by: https://github.com/jenkinsci/blueocean-plugin/pull/1299

**Pull request closed without merging**